IPython/nbconvert/filters/highlight.py

```diff
@@ -113,7 +113,7 @@ def _pygments_highlight(source, output_formatter, language='ipython', metadata=N
 
     # If the cell uses a magic extension language,
     # use the magic language instead.
-    if language == 'ipython' \
+    if language in ('ipython', 'ipython3') \
```

Comment:
The patched version works fine with python2:
 
![screen shot 2015-01-19 at 4 03 15 pm](https://cloud.githubusercontent.com/assets/535197/5807799/c09774cc-9ff4-11e4-992c-8fc8ae6ea370.png)